engine/src/test/java/org/hibernate/search/test/dsl/SortDSLTest.java

```diff
@@ -673,7 +682,7 @@ public void set(String name, Object value, Document document, LuceneOptions luce
 				return;
 			}
 
-			luceneOptions.addNumericFieldToDocument( name, doubleValue, document );
+			document.add( new DoubleDocValuesField( name, doubleValue ) );
```

Comment:
It's not strictly necessary, but as you wish. I amended the commit.

About making it easier, I created the ticket: https://hibernate.atlassian.net/browse/HSEARCH-2447